iaso/migrations/0328_auto_20250619_1436.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.20 on 2025-06-19 14:36
+
+from django.db import migrations
+from django.db.models.expressions import RawSQL
+
+
+def migrate_data_forward(apps, schema_editor):
+    chunk_size = 10000
+    Instance = apps.get_model("iaso", "Instance")
+
+    instances = (
+        Instance.objects.filter(form_version__isnull=True, json__isnull=False)
+        .annotate(
+            annotated_form_version_id=RawSQL(
+                "select id from iaso_formversion where version_id = iaso_instance.json->>'_version' limit 1", ()
+            )
+        )
+        .iterator(chunk_size=chunk_size)
+    )
+
+    instance_to_update = []
+    for instance in instances:
+        if not instance.annotated_form_version_id:
+            continue
+        instance.form_version_id = instance.annotated_form_version_id
+        instance_to_update.append(instance)
+        if len(instance_to_update) > chunk_size:
```

Comment:
```suggestion
        if len(instance_to_update) >= chunk_size:
```

I'm not sure if this is going to improve anything, but with `>` bulk updates will be triggered once we have 10001 elements, which means fetching 2 chunks from the iterator instead of 1